tests/base/model_train_steps.py

```diff
@@ -62,90 +60,23 @@ def training_step__inf_loss(self, batch, batch_idx, optimizer_idx=None):
                 output /= 0
         return output
 
-    def training_step_result_obj_dp(self, batch, batch_idx, optimizer_idx=None):
+    def training_step__result_obj_dp(self, batch, batch_idx, optimizer_idx=None):
+        self.training_step_called = True
```

Comment:
Is adding this necessary?

Reply:
it is just renamed to follow the guideline for tests...

Reply:
I mean adding `self.training_step_called = True`